General info about Centreon Cloud (#3511)
centreon/centreon-documentation: cloud/getting-started/about-centreon-cloud.md

---
id: about-centreon-cloud
title: About Centreon Cloud
---

## What is Centreon Cloud?

> Centreon Cloud is the Centreon monitoring platform installed in our Cloud infrastructure: read our [product description](https://www.centreon.com/centreon-cloud/).

Reviewer: ```suggestion
Centreon Cloud is the Centreon monitoring platform installed in our Cloud infrastructure. For detailed product information, visit our [product description page](https://www.centreon.com/centreon-cloud/).
```

## What is the SLA for Centreon Cloud?

> 

Reviewer: ```suggestion

Centreon Cloud is committed to a high service quality with the following guarantees:
```

* The SLA for Centreon Cloud is 99.5%. More information in our [Terms and Conditions](https://www.centreon.com/legal/en/Cloud-Services-terms-ROW).

* The RPO (Recovery Point Objective) is 24 hours.

* The RTO (Recovery Time Objective) is 4 hours.

## Where is my data hosted?

Centreon uses [several hosting providers](../security/security.md#hosting-security), mainly in Europe. We are [AWS partners](https://www.centreon.com/partners/centreon-on-aws/).

## How long is my data kept for?

Monitoring data is kept for 13 months: performance data (metrics), downtimes, acknowledgements, comments, logs...
All other data (e.g. your configuration) is kept until the end of your subscription.

## What security measures does Centreon take?

> Everything is explained in our [Centreon Cloud security management](../security/security.md) page.

Reviewer: ```suggestion
For a comprehensive overview of our security practices, visit our [Centreon Cloud security management page](../security/security.md).
```

## Fair use policy

Read our [Fair Use Policy](https://www.centreon.com/legal/en/centreon-cloud-service-fair-use-policy).

## Maintenance updates and upgrades

> * Your central server is updated and upgraded by Centreon, you have nothing to do.

Reviewer: ```suggestion
* Your central server is fully managed and updated by Centreon. No action is required on your part.
```

---
id: about-centreon-cloud
title: About Centreon Cloud
---

## What is Centreon Cloud?

> Centreon Cloud is the Centreon monitoring platform installed in our Cloud infrastructure. For detailed product information, visit our [product description page](https://www.centreon.com/centreon-cloud/).

Reviewer 2: Should rather be: 
Centreon Cloud is the Centreon monitoring platform delivered as a service and hosted and managed by Centreon from our Cloud infrastructure.

## What is the SLA for Centreon Cloud?

> Centreon Cloud is committed to a high service quality with the following guarantees:

Author: ```suggestion
Centreon Cloud is committed to a high quality of service with the following guarantees:
```

* The SLA for Centreon Cloud is 99.5%. More information in our [Terms and Conditions](https://www.centreon.com/legal/en/Cloud-Services-terms-ROW).

* The RPO (Recovery Point Objective) is 24 hours.

* The RTO (Recovery Time Objective) is 4 hours.

## Where is my data hosted?

Centreon uses [several hosting providers](../security/security.md#hosting-security), mainly in Europe. We are [AWS partners](https://www.centreon.com/partners/centreon-on-aws/).

## How long is my data kept for?

Monitoring data is kept for 13 months: performance data (metrics), downtimes, acknowledgements, comments, logs...
All other data (e.g. your configuration) is kept until the end of your subscription.

## What security measures does Centreon take?

For a comprehensive overview of our security practices, visit our [Centreon Cloud security management page](../security/security.md).

## Fair use policy

> Read our [Fair Use Policy](https://www.centreon.com/legal/en/centreon-cloud-service-fair-use-policy).

Author: ```suggestion
To understand our limitations in terms of resource use, read our [Fair Use Policy](https://www.centreon.com/legal/en/centreon-cloud-service-fair-use-policy).
```

## Maintenance updates and upgrades

> * Your central server is fully managed and updated by Centreon. No action is required on your part.

Reviewer 2: Should rather be:
The web application, data ingestion, transformation, and storage are fully managed [...]

> * Your pollers are in your infrastructure: it is your responsibility to [update or upgrade them](../installation/poller-update-upgrade.md).

Reviewer 2: Can you add a link to the glossary for "pollers"?
Also "it is your responsibility to install and keep them up-to-date"

> * Your pollers are in your infrastructure: it is your responsibility to [update or upgrade them](../installation/poller-update-upgrade.md).

Author: ```suggestion
* Your [pollers](../resources/glossary.md#poller) are in your infrastructure: it is your responsibility to [update or upgrade them](../installation/poller-update-upgrade.md).
```
